### ISSUE TITLE
Can't check capabilities on Android Chrome

### ISSUE THREAD

**Issue by markheloking** [theotherp/nzbhydra2#324]
On Chrome on Android phone, the interface for configuring an indexer is very narrow, breaking up the seperation between the tags and the fields (see screenshot). Even though I can see the "check capabilities" refresh button, it doesn't seem to be a clickable UI element. The other fields and dropdown boxes do work as expected. I'm suspecting the "action" overlay of the button might be off (not actually positioned over the UI graphical button), but I'm not sure that's how that works.

Happens both in normal mode and in "desktop mode".

Screenshots:
![nzbhydra2_ui_broken1](https://user-images.githubusercontent.com/3192761/52906118-09801800-3246-11e9-87eb-3cfccccdf7de.jpeg)
![nzbhydra2_ui_broken2](https://user-images.githubusercontent.com/3192761/52906119-0a18ae80-3246-11e9-89a7-1ee800358a77.jpeg)

Since this is a UI bug, there's no log to be attached, sorry.

NZBHydra2 2.3.15 running in Docker on Windows.
Android 9, latest chrome build.

The problem has persisted for as long as I've been using NZBHydra2 (a few months now) over all versions.

TL;DR: Can't click the check capabilities button on Chrome on Android,  works fine on desktop.

[EDIT]
Also doesn't work on Firefox on Android.

**Comment by theotherp** [comment]
Yeah, sorry. I won't go through the pains of making it work on mobile.
